packages/fether-react/package.json

```diff
@@ -31,7 +31,7 @@
     "start": "npm-run-all -p start-*",
     "start-css": "npm run build-css -- --watch --recursive",
     "start-js": "cross-env SKIP_PREFLIGHT_CHECK=true BROWSER=none craco start --react-scripts ../../node_modules/react-scripts",
-    "test": "ln -s ../../../node_modules/react-scripts node_modules/ && cross-env SKIP_PREFLIGHT_CHECK=true craco test; rm node_modules/react-scripts"
+    "test": "ln -sf ../../../node_modules/react-scripts node_modules/ && cross-env SKIP_PREFLIGHT_CHECK=true craco test"
```

Comment:
i think this should be `ln -sf ../../node_modules` instead of `ln -sf ../../../node_modules`

Reply:
I thought so too, but it's actually relative to `node_modules/` itself, so it should be `../../../`. Just ran on my bash to confirm

Reply:
I would keep the `rm node_modules/react-scripts`. We're "creating" this folder manually purely for the test, it has no reason to stay in node_modules/ once it's over (https://github.com/paritytech/fether/pull/284#issuecomment-447403221). I'm afraid it could interfere with yarn at some point if we keep the folder.

Reply:
Agreed. The only reason I removed it is because when tests fail, this command still has an exit code 0.

The ideal scenario would be:
- `ln -s ...`
- if tests pass, return 0, and rm the symlink
- if test don't pass, return 1, and rm the symlink

But I don't know how to do that in bash. The current return code of the command is the `rm symlink`, so always 0, which is incorrect

Reply:
I think this should work:

```bash
ln -s ../../../node_modules/react-scripts node_modules/ && cross-env SKIP_PREFLIGHT_CHECK=true craco test; EXIT_CODE=$?; rm node_modules/react-scripts; exit $EXIT_CODE
```

Reply:
nice 😎

Reply:
LGTM